packages/styletron-standard/src/style-types.ts

```diff
@@ -0,0 +1,43 @@
+import type {
+  StandardProperties,
+  VendorProperties,
+  ObsoleteProperties,
+  SvgProperties,
+  AnimationNameProperty as CTAnimationNameProperty,
+  FontFamilyProperty as CTFontFamilyProperty,
+  FontFace as CTFontFace,
+} from "csstype";
+
+export interface KeyframesPercentageObject {
+  [key: string]: Properties;
+}
+
+export type KeyframesObject = KeyframesPercentageObject & {
+  from?: Properties;
+  to?: Properties;
+};
+
+export type AnimationNameProperty = CTAnimationNameProperty | KeyframesObject;
+
+export type FontFace = CTFontFace;
+
+export type FontFamilyProperty = CTFontFamilyProperty | FontFace;
+
+export type TLength = string | 0;
+export type Properties = {
+  animationName?: AnimationNameProperty;
+  fontFamily?: FontFamilyProperty | FontFamilyProperty[];
+  MozAnimationName?: AnimationNameProperty;
+  WebkitAnimationName?: AnimationNameProperty;
+  OAnimationName?: AnimationNameProperty;
+} & Omit<
```

Comment:
This `Omit` utility type is ending up in the flow type definitions generated. See published file here: https://unpkg.com/@miksu/styletron-standard@0.0.1/dist-node-cjs/index.js.flow

Another issue in the generated file is that import statements do not include the `type` prefix.

```
import { AnimationNameProperty as AnimationNameProperty_2 } from "csstype";
// should be
import type { AnimationNameProperty as AnimationNameProperty_2 } from "csstype";
```

Reply:
I suspect flowgen is not yet supporting `import type` typescript feature (which was added relatively recently), same about `Omit` which is older but still probably new comparing to most of flowgen code